### ISSUE TITLE
Add improvement discussion repository

### ISSUE THREAD

**Issue by Dekoze** [clemahieu/raiblocks#570]
Similar to BIPs and EIPs (maybe don't call them NIPs) I think we should start having a place for protocol discussion and higher impact changes separate from the code. I think valuable information will not get proper exposure here due to the noisy nature of a main source repo. A new process doesn't have to be enforced but centralizing important discussions to a space specifically for them is beneficial for stakeholders.

In my view, a PR like #554 would benefit from this since there is discussion of implementation mixed with higher level impact that can be distilled.

**Comment by peteratticusberg** [comment]
I agree. As a developer considering getting involved the project, it would be helpful to have a place a go to understand what kinds of improvement proposals have been considered, which are currently being considered and what the consensus the team has reached on these things. 

For example, I'd love to be able to read about the team's thoughts on rate limiting and account minimums, topics I'm sure have been discussed extensively, but perhaps not in a discoverable forum (AFAICT).

I'd be happy to run point on setting something like this up, using the documentation around process for https://github.com/bitcoin/bips as a starting point and sourcing feedback in this thread from the team on if/where we'd like to modify it for nano.
